gomock/controller.go

```diff
@@ -296,3 +296,9 @@ func unwrapTestReporter(t TestReporter) TestReporter {
 	}
 	return tr
 }
+
+// AllExpectedCallsSatisfied returns true in case all expected calls bound to this Controller are satisfied.
+// Calling Finish is then guaranteed to not fail due to missing calls.
+func (ctrl *Controller) AllExpectedCallsSatisfied() bool {
```

Comment:
Same thing here as above 
```suggestion
func (ctrl *Controller) Satisfied() bool {
```

Reply:
👍